public/video-ui/src/components/Header.jsx

```diff
@@ -85,8 +85,9 @@ export default class Header extends React.Component {
             );
           }}
         >
-          <option value="false">All videos</option>
-          <option value="true">Loops</option>
+          <option value="">All videos</option>
+          <option value="url">Loops</option>
+          <option value="youtube">YouTube</option>
```

Comment:
We could extend to other media platform types (https://github.com/guardian/content-atom/blob/main/thrift/src/main/thrift/atoms/media.thrift#L10-L16) but I'm not sure there's much appetite for this at present. Would be trivial to add if we wanted to, now this code is extensible.